src/main/scala/exercises/07_Palindrome.scala

```diff
@@ -8,6 +8,8 @@ object Palindrome {
    *  Hint: the Scala collections API is your friend
    *  http://docs.scala-lang.org/overviews/collections/overview.html 
    */
-  def isPalindrome(s: String): Boolean = throw new NotImplementedError
-
+  def isPalindrome(s: String): Boolean = {
+  	val onlyLetters = s.filter(_.isLetter).map(_.toLower)
+  	onlyLetters == onlyLetters.reverse
```

Comment:
Nice! Scala also has a built in comparison that ignores case so you could remove the `.map(_.toLower)`:

```scala
a.equalsIgnoreCase(b)
```